pallet/src/mock.rs

```diff
@@ -111,7 +112,6 @@ impl pallet_session::Config for Test {
     type SessionManager = pallet_aleph::AlephSessionManager<Self>;
     type SessionHandler = <TestSessionKeys as OpaqueKeys>::KeyTypeIdProviders;
     type Keys = TestSessionKeys;
-    type DisabledValidatorsThreshold = DisabledValidatorsThreshold;
```

Comment:
you can then remove parameter as well (~line 87)

Reply:
done